initdb.d/01_main.sql

```diff
@@ -1,339 +1,143 @@
--- \connect lega
+CREATE SCHEMA sda;
 
--- We use schemas for isolation instead of multiple databases.
--- Look at the grants.sql file, for access user/role rights setup
+SET search_path TO sda;
 
-CREATE SCHEMA local_ega; -- includes the main table, some views and errors
-
-SET search_path TO local_ega;
-
-CREATE TYPE checksum_algorithm AS ENUM ('MD5', 'SHA256', 'SHA384', 'SHA512'); -- md5 is bad. Use sha*!
+-- ENUMS
+CREATE TYPE checksum_algorithm AS ENUM ('MD5', 'SHA256', 'SHA384', 'SHA512');
+CREATE TYPE encryption_algorithm AS ENUM ('CRYPT4GH', 'PGP', 'AES');
 CREATE TYPE storage AS ENUM ('S3', 'POSIX');
--- Note: This is an enum, because that's what the "provided" database supports
---       If a site has its own database already, let them define their keyword in the ENUM 
---       and use it (Notice that their code must be update to push this value into the table)
---       There is no need to agree on how each site should operate their own database
---       What we need is to document where they need to update and what.
+CREATE TYPE file_state AS ENUM ('UPLOADED', 'ARCHIVED', 'UNENCRYPTED');
```

Comment:
this might not be clear in its name and usage, currently we only use it for `checksums` so maybe `checksum_source`, or adding more information on its usage would be ideal

Reply:
good point! `checksum_source` is good!